Library/Sender.ts

```diff
@@ -160,7 +162,7 @@ class Sender {
     private _storeToDisk(payload: any) {
 
         //ensure directory is created
-        var directory = path.join(os.tmpdir(), Sender.TEMPDIR);
+        var directory = path.join(os.tmpdir(), Sender.TEMPDIR_PREFIX + this._config.instrumentationKey);
```

Comment:
make sure it's not an information disclosure. Multiple processes may have access to temp folder which is sometimes shared among multiple processes. Maybe take a hash of it?

Reply:
To clarify, are you suggesting hashing the ikey in the folder name, so that other processes may not read it? We could do so, however even if we take the hash of the folder name, thus hiding the ikey, what's preventing the other process from parsing the data in the folder, where ikey is part of every envelope? - in fact they can do so today.

Reply:
ok. good point. Perhaps allow to override the path so one worried about privacy and tampering can configure it to ACL-ed directory.

Reply:
Added #307